### DISCUSSION TITLE
Access Denied erreur when using winSCP to list bucket contenent

### DISCUSSION THREAD

**Discussion by Amine-741** [minio/minio#17941] [Q&A] [answered]
Hello everybody,

I have an issue with MinIO.
I set à policy to give access to specific subfolder to specific user , this is the policy that i gave 

`{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "s3:ListBucket"
            ],
            "Resource": [
                "arn:aws:s3:::bucket-name"
            ],
            "Condition": {
                "StringEquals": {
                    "s3:delimiter": [
                        "/"
                    ],
                    "s3:prefix": [
                        ""
                    ]
                }
            }
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:ListBucket"
            ],
            "Resource": [
                "arn:aws:s3:::bucket"
            ],
            "Condition": {
                "StringLike": {
                    "s3:prefix": [
                        "Folder1/Folder2/*"
                    ]
                }
            }
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:*"
            ],
            "Resource": [
                "arn:aws:s3:::bucket-name/Folder1/Folder2/*"
            ]
        }
    ]
}`

From minio web console i can see that the user has access to this specific folder2, but when i try to access from winSCP i get Access Denied erreur. Even when using mc command.

PS: i can see all folders at / , but i can't access any one of them even for those which i set to policy

Could you help me please .

Thank you

**Comment by harshavardhana** [comment, marked as answer]
You should look at what other S3 calls winscp does.

If Console UI works, mc works then winscp is expecting more permissions here.

**Comment by Amine-741** [comment]
Even with mc is not working 
mc: <ERROR> Unable to list folder. Access Denied.
